pkg/oc/bootstrap/docker/up.go

```diff
@@ -260,29 +259,29 @@ func (c *CommonStartConfig) addTask(t task) {
 	c.Tasks = append(c.Tasks, t)
 }
 
-func (config *CommonStartConfig) Bind(flags *pflag.FlagSet) {
```

Comment:
the only change here is the s/config/c/ (golint complains about receivers being different and i like less typing anyway)